.github/workflows/no-response.yml

```
@@ -24,8 +24,8 @@ jobs:
           # Number of days of inactivity before an Issue is closed for lack of response
           daysUntilClose: 7
           # Label requiring a response
-          responseRequiredLabel: 'status: needs more information'
+          responseRequiredLabel: 'status: waiting for follow-up'
           # Comment to post when closing an Issue for lack of response. Set to `false` to disable
           closeComment: >
-            Since the issue is missing key information and has been inactive for 7 days, it has been automatically closed.
-            If you wish to see the issue reopened, please provide the missing information.
+            The issue has been inactive for 7 days and is now automatically closed.
```

Maybe keep the same tense as the start of the sentence?
```suggestion
            The issue has been inactive for 7 days and has been automatically closed.
```